Content.Server/Station/Systems/StationSpawningSystem.cs

```diff
@@ -175,6 +175,24 @@ public EntityUid SpawnPlayerMob(
 
         SetupCybernetics(entity.Value, profile?.Cybernetics ?? []); // Starlight
 
+        // Starlight begin
+        if (loadout != null && prototype?.StartingGear != null)
+        {
+            var startingGear = _prototypeManager.Index<StartingGearPrototype>(prototype.StartingGear);
+            StarlightEquipRoleLoadout(entity.Value, loadout, [startingGear], roleProto!);
+        }
+        else if (loadout != null)
+        {
+            StarlightEquipRoleLoadout(entity.Value, loadout, [], roleProto!);
+        }
+        else if (prototype?.StartingGear != null)
+        {
+            var startingGear = _prototypeManager.Index<StartingGearPrototype>(prototype.StartingGear);
+            EquipStartingGear(entity.Value, startingGear, raiseEvent: false);
+        }
```

Comment:
you can simplify this


```suggestion
        if (loadout != null)
        {
            var startingGear = prototype?.StartingGear != null ? [_prototypeManager.Index<StartingGearPrototype>(prototype.StartingGear)] : [];
            StarlightEquipRoleLoadout(entity.Value, loadout, startingGear, roleProto!);
        }
        else if (prototype?.StartingGear != null)
        {
            var startingGear = _prototypeManager.Index<StartingGearPrototype>(prototype.StartingGear);
            EquipStartingGear(entity.Value, startingGear, raiseEvent: false);
        }
```